SC_redesign/Crop_and_Soil/soil/soil.py

```diff
@@ -8,5 +8,9 @@ def __init__(self):
         self.lower_boundaries = [5, 8, 20]
         self.nitrates = [0.5, 1, 5]
 
+        #attrs for phosphorus_uptake
+        self.labile_phosphorus = []
+        self.phosphorus_uptake = []
```

Comment:
I'm not positive about this, but isn't there something in python where starting with a default that is a mutable object leads to bugs? I think its better to start with some default values. Also, I think it is important that the length of all layer-specific variables are equal. 
```suggestion
        #attrs for phosphorus_uptake
        self.labile_phosphorus = [3, 6, 9]
        self.phosphorus_uptake = [0, 0, 0]
```